tests/php/modules/videopress/test_functions.utility-functions.php

```diff
@@ -12,6 +12,181 @@
  */
 class WP_Test_Jetpack_VideoPress_Utility_Functions extends WP_UnitTestCase {
 
+	/**
+	 * Tests a helper function to get the post by guid, when there is no post found.
+	 *
+	 * @covers ::video_get_post_by_guid
+	 * @since 8.4.0
+	 */
+	public function test_no_post_found_video_get_post_by_guid() {
+		$this->assertFalse( video_get_post_by_guid( wp_generate_uuid4() ) );
+	}
+
+	/**
+	 * Gets the test data for test_cached_video_get_post_by_guid().
```

Comment:
minor: I think you meant `test_non_cached_video_get_post_by_guid`

Reply:
Sure, good catch. Applied in https://github.com/Automattic/jetpack/pull/14803/commits/5c05e33d54e8a7987024dc3c7891f54ce6c5a572